BitwardenShared/UI/Auth/Login/TwoFactorAuth/TwoFactorAuthProcessor.swift

```diff
@@ -377,7 +377,10 @@ extension TwoFactorAuthProcessor: DuoAuthenticationFlowDelegate {
 
         guard let authURLValue = maybeAuthURL,
               let authURL = URL(string: authURLValue) else {
-            state.toast = Toast(text: Localizations.duoUnsupported)
+            state.toast = Toast(
+                // swiftlint:disable:next line_length
+                text: Localizations.errorConnectingWithTheDuoServiceUseADifferentTwoStepLoginMethodOrContactDuoForAssistance
+            )
```

Comment:
🤔 Can we add a test for this?
🤔 I see the localization string `Localizations.duoUnsupported` is not used anywhere else. Can we remove it from the localizations file?